includes/kco-functions.php

```diff
@@ -97,6 +97,7 @@ function kco_wc_show_snippet( $pay_for_order = false ) {
 
 	if ( isset( $klarna_order['html_snippet'] ) ) {
 		do_action( 'kco_wc_show_snippet', $klarna_order );
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- We trust the HTML snippet.
```

Comment:
Should this not be on the same line as the echo for it to work properly?

Reply:
It will ignore the current and next line: https://stackoverflow.com/a/59073074